README.md

```diff
@@ -498,6 +500,26 @@ Will produce the following payload:
 }
 ```
 
+## Sorting lists by multiple columns
+
+Hasura support [sorting by multiple fields](https://hasura.io/docs/latest/graphql/core/databases/postgres/queries/sorting.html#sorting-by-multiple-fields) but React Admin itself doesn't allow the `List` component to receive an array as the `sort` prop. So to achieve sorting by multiple fields, separate the field and order values using a comma.
+
+For example, a list like
+
+```jsx
+const TodoList = (props) => (
+  <List sort={{ field: 'title,is_completed', order: 'asc,desc' }} {...props}>
```

Comment:
Can we instead have a ` { field: 'field_name@order_type,field_name@order_type }` format so that:

- its more readable (co-locating the field_name and its order_type)
- it follows the syntax format used for filtering via multiple fields (`column_name@filter_type, column_name@filter_type`)

Reply:
Unfortunately, we can't. The React-Admin `<List />` component requires a `sort` prop object of type
```
type Sort = { field: string, order: string }
```
The multiple fields filtering was just a string prop so we could pass whatever we wanted i.e. the `@` syntax.

See https://marmelab.com/react-admin/List.html#sort-default-sort-field--order